__tests__/api.test.js

```diff
@@ -53,3 +53,40 @@ describe("GET/api", () => {
       });
   });
 });
+describe("GET/api/articles/:article_id", () => {
+  it("status: 200, responds with the article with relevant id with the correct data type", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.article).toBe("object");
```

Comment:
This test is not robust enough. If the server were to send me back article 20 this test would pass. At the very least you should be asserting that the article_id is 3 as that is the request you have made.

Reply:
Line 62, what data type is an array? 
By checking body.article and toHaveProperty you have already asserted that this is an object as arrays cannot be accessed in this manner.